src/lib/nftClaimer/utils.ts

```diff
@@ -132,3 +136,34 @@ export function validateAddresses(addresses: Record<string, string>) {
 
   return true;
 }
+
+export async function snapshotFee(force = false) {
+  const storage = storageEngine('nft-claimer');
+  const keyname = 'snapshotFee';
+  const CACHE_TTL = 60 * 60 * 1000; // 1 hour
+
+  try {
+    const cache = await storage.get(keyname);
+    if (typeof cache === 'string') {
+      const result = JSON.parse(cache);
+
+      if (result.lastUpdate > +new Date() - CACHE_TTL && !force) {
+        return result.snapshotFee;
+      }
+    }
+
+    const provider = snapshot.utils.getProvider(NFT_CLAIMER_NETWORK);
+    const contract = new Contract(
+      DEPLOY_CONTRACT,
+      ['function snapshotFee() public view returns (uint8)'],
+      provider
+    );
```

Comment:
Also we can move these out of this function as they will be always the same

Reply:
Cache removed